plugin/src/sbt-test/psv/custom_sbt_ivy_home_config/test

```diff
@@ -0,0 +1,5 @@
+> set scriptedBufferLog := false
```

Comment:
Why cannot we combine the test with these tests? 

https://github.com/heremaps/here-sbt-bom/blob/master/plugin/src/sbt-test/psv/custom_ivy_home_config/test

Reply:
We can. No objections.